lib/build_targets/package.dart

```diff
@@ -285,7 +286,10 @@ class NativeTpk extends TizenPackage {
       embeddingDir.path,
       configuration: buildConfig,
       arch: getTizenCliArch(buildInfo.targetArch),
-      extraOptions: <String>['-fPIC'],
+      extraOptions: <String>[
+        '-fPIC',
+        '-D${buildInfo.deviceProfile.toUpperCase()}_PROFILE'
```

Comment:
Please use

```dart
        predefines: <String>[
          '${buildInfo.deviceProfile.toUpperCase()}_PROFILE',
        ],
```

for consistency. (Please refer to `plugins.dart` for the exact order between the arguments.)

Reply:
oh thanks. i just understanded this out